### PR TITLE
Add heiko crowdloan, rate for INTR

### PR DESCRIPTION
**Pull request by leohar**
- Heiko Crowdloan
[source](https://parallelfi.gitbook.io/parallel-finance/polkadot-network/parachain-crowdloans/parallel-heiko-crowdloan/faq#what-are-the-hko-reward-distributions)

- Interlay rate
[INTR on Coingecko](https://www.coingecko.com/en/coins/interlay)
[rate source](https://docs.google.com/spreadsheets/d/1wTTuxXt8n9q7C4NDXqQpI3wpKu1_5bGVmP9Xz0XGSyU/edit#gid=0)

### CODE REVIEW

File: crowdloan/kusama.json [hunk 1]
```diff
@@ -418,5 +418,14 @@
     "description": "Amplitude is aiming to provide novel ways of utilizing fiat on DeFi",
     "website": "https://pendulumchain.org/amplitude",
     "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Amplitude.svg"
+  },
+  {
+    "paraid": "2126",
+    "name": "Parallel Heiko",
```
Comment (antonkhvorov) [inline crowdloan/kusama.json:424]: Parallel Heiko 2

File: crowdloan/kusama.json [hunk 2]
```diff
@@ -418,5 +418,14 @@
...
+    "token": "HKO",
+    "description": "Decentralized money market protocol that offers lending, staking, and borrowing",
+    "website": "https://parallel.fi/index.html",
+    "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/ParallelHeiko.svg",
+    "rewardRate": 200
```
Comment (antonkhvorov) [inline crowdloan/kusama.json:429]: Probably better to remove it

File: crowdloan/kusama.json [hunk 3]
```diff
@@ -418,5 +418,14 @@
     "description": "Amplitude is aiming to provide novel ways of utilizing fiat on DeFi",
     "website": "https://pendulumchain.org/amplitude",
     "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Amplitude.svg"
+  },
+  {
+    "paraid": "2126",
+    "name": "Parallel Heiko",
+    "token": "HKO",
+    "description": "Decentralized money market protocol that offers lending, staking, and borrowing",
+    "website": "https://parallel.fi/index.html",
```
Comment (antonkhvorov) [inline crowdloan/kusama.json:427]: https://parallel.fi/

**Review by antonkhvorov — APPROVED** [review]

File: crowdloan/kusama.json (hunk 3 above)
Comment (leohar) [reply crowdloan/kusama.json:427]: done

**Pull request merged**